src/FieldWarning/Assets/Standard Assets/3DParty/TextMesh Pro/Resources/Fonts & Materials/LiberationSans SDF.asset

```diff
@@ -138,7 +138,7 @@ Material:
     - _Color: {r: 1, g: 1, b: 1, a: 1}
     - _EmissionColor: {r: 0, g: 0, b: 0, a: 1}
     - _EnvMatrixRotation: {r: 0, g: 0, b: 0, a: 0}
-    - _FaceColor: {r: 1, g: 1, b: 1, a: 1}
+    - _FaceColor: {r: 0, g: 0, b: 0, a: 1}
     - _GlowColor: {r: 0, g: 1, b: 0, a: 0.5}
```

Comment:
What's the effect of this?

Reply:
no idea.. i thought this was from my pull from master... Maybe its a bold setting?